backend/src/routes/api/cluster-settings/clusterSettingsUtils.ts

```diff
@@ -1,17 +1,21 @@
 import { FastifyRequest } from 'fastify';
+import { scaleDeploymentConfig } from '../../../utils/deployment';
 import { KubeFastifyInstance, ClusterSettings } from '../../../types';
 
+const juypterhubCfg = 'jupyterhub-cfg';
+const setgmentKeyCfg = 'rhods-segment-key-config';
```

Comment:
`setgmentKeyCfg` is the wrong spelling :sweat_smile: 
Should be `segmentKeyCfg`

Reply:
done.